web/src/__tests__/mocks/supabase-db.ts

```
@@ -0,0 +1,298 @@
+            const data = queryState.single ? result[0] : result;
+            resetQueryState();
+            resolve({ data, error: null });
+          };
+
+          return chain;
+        }),
+
+        delete: vi.fn(() => {
+          const chain = createQueryResult('delete');
+
+          chain.then = (resolve: (result: { data: unknown; error: unknown; count: number }) => void) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)